### ISSUE TITLE
Bug in the Email validator

### ISSUE THREAD

**Issue by solventt** [rakit/validation#143]
For example, there are validation rules for the email field: ```'email' => 'required|string|email|unique:emails,email'```

This way causes that only the message of the 'email' rule to be always shown whatever rule is triggered.

But if I change the validator name other than 'email' - the problem goes away, e.g.:
```'other' => 'required|string|email|unique:emails,email' ```

**Comment by ookamiiixd** [comment]
I faced the same issue.

**Comment by michaelthedev** [comment]
Currently facing the same issue

**Comment by michaelthedev** [comment]
You've probably carried on with your lives on this haha

But for anyone having the same issue, i did this fix [https://github.com/rakit/validation/issues/161#issue-2223690913](https://github.com/rakit/validation/issues/161#issue-2223690913)
